astropy/modeling/functional_models.py

```diff
@@ -1432,9 +1432,9 @@ class Box2D(Fittable2DModel):
 
             f(x, y) = \\left \\{
                      \\begin{array}{ll}
-                       A & : x_0 - w_x/2 \\geq x \\geq x_0 + w_x/2 \\\\
-                       A & : y_0 - w_y/2 \\geq y \\geq y_0 + w_y/2 \\\\
-                       0 & : \\textnormal{else}
+                       A & : x_0 - w_x/2 \\leq x \\leq x_0 + w_x/2 \\\\
+                       A & : y_0 - w_y/2 \\leq y \\leq y_0 + w_y/2 \\\\
```

Comment:
Taken from Spyder (no time to build docs again right now):

![unbenannt](https://cloud.githubusercontent.com/assets/14200878/18177591/2884c6cc-707b-11e6-97a4-6c164c15e6bd.jpg)